### PR TITLE
docs(Buttons): first version

### PR DESCRIPTION
**Pull request by kibos11**
This is the first commit of this content (just a wiki mirror)

### CODE REVIEW

File: src/systems/ec/specs/components/button/docs/usage.md [hunk 1]
```diff
@@ -1 +1,28 @@
...
+
+### Button labels 
+Button labels tell users what will happen when they click the button. Use verbs that describe the action, such as Send, Add or Delete. Do not use capitalization on buttons (only the first word in a phrase and proper nouns are capitalized) and no more than three words for button labels.
+
+### Button icons
+It is possible to add an icon to a button to enhance the action.
+
+| Button name           | Visual style  |
+| ----------------------|---------------|
+| Call-to-action button | <img src="http://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_CTA-Icon_Default.png" alt="Button Image" style="width:90;"/>  |
```
Comment (yhuard) [inline src/systems/ec/specs/components/button/docs/usage.md]: Please replace `style="width:90;"` by `width="90"`

File: src/systems/ec/specs/components/button/docs/usage.md [hunk 2]
```diff
@@ -1 +1,28 @@
-Do's and don'ts
+Use a button to trigger and carry out an action on a page. 
+
+## When to use
+The ECL has five buttons with different visual designs to make a distinction between the kind of action the button will carry out.
+
+| Button name           | Visual style  | Usage |
+| ----------------------|---------------| ------|
+| Call-to-action button | ![Button Image](http://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_CTA_Default.png) | for the most important action on the page like "Apply for funding" or "Watch live streaming".|
```
Comment (yhuard) [inline src/systems/ec/specs/components/button/docs/usage.md]: Please use https instead of http

**Comment by kibos11** [comment]
I did the changes. Do I need to do anything else or it will merge automatically at the time that you will review it? Also, I would like to ask about the step "_Do not forget to add tag: documentation on your pull request which is used for the release log"_. How can I do that?

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit de79f727f9b1bd2a8a9ed0848a8c576d4a68e102

https://deploy-preview-836--europa-component-library.netlify.com

**Pull request merged**